src/parallel/DomainDecompMPIBase.cpp

```diff
@@ -197,7 +195,7 @@ void DomainDecompMPIBase::assertIntIdentity(int IX) {
 			MPI_CHECK(MPI_Recv(&recv, 1, MPI_INT, i, 2 * i + 17, _comm, &s));
 			if (recv != IX) {
 				Log::global_log->error() << "IX is " << IX << " for rank 0, but " << recv << " for rank " << i << ".\n";
-				MPI_Abort(_comm, 911);
+				MPI_Abort(_comm, 911); // TODO FIXME
```

Comment:
@FG-TUM Do you know, why MPI_Abort is called directly here (and also at other locations)?

Reply:
No idea. Probably because the person didn't know about `MARDYN_EXIT()`. I'd suggest using our exit function / macro.